packages/tooling/fast-tooling-react/src/form/custom-controls/control.css.utilities.syntax.tsx

```diff
@@ -3,6 +3,7 @@ import { Syntax } from "@microsoft/fast-tooling/dist/css-data.syntax";
 import { syntaxes } from "@microsoft/fast-tooling/dist/css-data";
 import { RenderRefControlConfig } from "./control.css.utilities.props";
 import { CSSRef } from "@microsoft/fast-tooling-react/src/form/custom-controls/control.css-ref";
+import { renderColorPicker } from "@microsoft/fast-tooling-react/src/form/custom-controls/control.css.utilities";
```

Comment:
That top one is my bad, all intra-package dependencies should be relative.

```suggestion
import { CSSRef } from "./control.css-ref";
import { renderColorPicker } from "./control.css.utilities";
```

Reply:
Fixed.